lib/pyfrc/physics/drivetrains.py

```diff
@@ -176,7 +176,7 @@ def four_motor_swerve_drivetrain(lr_motor, rr_motor, lf_motor, rf_motor, lr_angl
     rf_rad = rf_angle * (math.pi / 180)
 
     # Calculate wheelbase radius
-    wheelbase_radius = math.sqrt((x_wheelbase ** 2) + (y_wheelbase ** 2))
+    wheelbase_radius = math.sqrt(((x_wheelbase/2) ** 2) + ((y_wheelbase/2) ** 2))
```

Comment:
You can get rid of two pairs of parentheses here. Order of operations should be obvious to anyone who's done… well, any maths.

(Seeing more than two consecutive open parentheses weirds me out. Maybe I'm just allergic to Lisp.)